src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_help.py

```diff
@@ -0,0 +1,124 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+# pylint: disable=line-too-long
```

Comment:
You don't need this in this file. The line length limitation is 120.

Reply:
done